lib/snapshot/options.rb

```diff
@@ -0,0 +1,142 @@
+        FastlaneCore::ConfigItem.new(key: :workspace,
+                                     short_option: "-w",
+                                     env_name: "SNAPSHOT_WORKSPACE",
+                                     optional: true,
+                                     description: "Path the workspace file",
+                                     verify_block: proc do |value|
+                                       v = File.expand_path(value.to_s)
+                                       raise "Workspace file not found at path '#{v}'".red unless File.exist?(v)
+                                       raise "Workspace file invalid".red unless File.directory?(v)
+                                       raise "Workspace file is not a workspace, must end with .xcworkspace".red unless v.include?(".xcworkspace")
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :project,
+                                     short_option: "-p",
+                                     optional: true,
+                                     env_name: "SNAPSHOT_PROJECT",
+                                     description: "Path the project file",
+                                     verify_block: proc do |value|
+                                       v = File.expand_path(value.to_s)
+                                       raise "Project file not found at path '#{v}'".red unless File.exist?(v)
+                                       raise "Project file invalid".red unless File.directory?(v)
+                                       raise "Project file is not a project file, must end with .xcodeproj".red unless v.include?(".xcodeproj")
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :devices,
+                                     description: "A list of devices you want to take the screenshots from",
+                                     is_string: false,
+                                     optional: true),
+                                     # verify_block: Proc.new do |value|
+                                     #   raise "Devices must be an array" unless value.kind_of?Array
+                                     #   available = Simulators.available_devices(name_only: true)
+                                     #   value.each do |current|
+                                     #     unless available.include?current
+                                     #       raise "Device '#{current}' not in list of avaiable simulators '#{available.join(', ')}'"
+                                     #     end
+                                     #   end
+                                     # end),
+        FastlaneCore::ConfigItem.new(key: :languages,
+                                     description: "A list of languages which should be used",
+                                     is_string: false,
+                                     default_value: [
+                                      'en-US',
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.